docs/root/terminology.md

```diff
@@ -0,0 +1,40 @@
+# Terminology
+
+## Blocking
+
+In [closed-loop](#closed-loop) mode, Nighthawk may delay
+[request-release-timings](#request-release-timing) because it needs to wait on
+the server. We call this blocking, and when that happens it gets reported in a histogram.
+
+## Closed-loop
+
+In closed-loop mode, there is a feedback loop between the test subject and the
+test system. Concretely, Nighthawk will track and report [blocking](#blocking).
+When this happens, the current linear rate limiter will attempt to compensate
+pacing.
+
+When significant blocking is observed, measurements are skewed. Therefore,
+open-loop testing should be considered the preferred mode when measuring latency.
+
+Closed-loop testing can be useful to get a sense of the maximum requests per
+second that can be squeezed out of a system, as well as for comparing numbers
```

Comment:
I would suggest we add: "...that can be squeezed out of a system irrespective of the high latency incurred."

Or something similar to point out that the resulting measurement might not be indicative of the maximum load a system can reasonably handle in production without customer visible impact.